apps/vessel-history/features/profile/components/GuideAndVideo.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { event as uaEvent } from 'react-ga'
+import { useTranslation } from 'react-i18next'
+import styles from './Info.module.css'
+
+
+interface GuideAndVideoProps {
+    source?: string
```

Comment:
Why not using `label` for this prop, it seems more straight forward? What do you think?

Reply:
because I was thinking the source of where is clicked